store.go

```diff
@@ -60,12 +60,9 @@ func (c *controller) getNetworkFromStore(nid string) (*network, error) {
 	for _, store := range c.getStores() {
 		n := &network{id: nid, ctrlr: c}
 		err := store.GetObject(datastore.Key(n.Key()...), n)
-		if err != nil && err != datastore.ErrKeyNotFound {
-			return nil, fmt.Errorf("could not find network %s: %v", nid, err)
-		}
-
 		// Continue searching in the next store if the key is not found in this store
-		if err == datastore.ErrKeyNotFound {
+		if err != nil {
+			log.Debugf("could not find network %s: %v", nid, err)
```

Comment:
We should only log the debug, only if the error is not `datastore.ErrKeyNotFound`. `datastore.ErrKeyNotFound` is a pretty common case so the debug logs will have a lot of unnecessary noise.